shared/uavobjectdefinition/stabilizationsettings.xml

```diff
@@ -30,7 +30,7 @@
 	<field name="VbarPiroComp" units="" type="enum" elements="1" options="FALSE,TRUE" defaultvalue="FALSE"/>
 	<field name="VbarMaxAngle" units="deg" type="uint8" elements="1" defaultvalue="10"/>
 
-	<field name="GyroTau" units="" type="float" elements="1" defaultvalue="0.005"/>
+	<field name="GyroCutoff" units="Hz" type="float" elements="1" defaultvalue="40.6"/>

```

Comment:
What are your assumptions on this value?  This configured to be the same for how many Hz of loop rate?

I think we should pick a compromise value that is higher than the default with the lower loop rates, but lower than the default with the higher rates that some people run.  I also would prefer a round number, like 45.